### PR TITLE
Init 2.387.1

### PR DESCRIPTION
**Pull request by NotMyFault**

### CODE REVIEW

File: profile.d/stable [hunk 1]
```diff
@@ -2,7 +2,7 @@
 # WARNING: Any variables defined here, override those defined from the Jenkinsfile
 #
 #
-RELEASE_GIT_BRANCH=master
+RELEASE_GIT_BRANCH=2.387
```
Comment (timja) [inline profile.d/stable:5]:
```suggestion
RELEASE_GIT_BRANCH=stable-2.387
```

File: Jenkinsfile.d/core/package [hunk 2]
```diff
@@ -83,7 +83,7 @@ pipeline {
     GPG_FILE                  = 'jenkins-release.gpg'
     GPG_PASSPHRASE            = credentials('release-gpg-passphrase')
     PACKAGING_GIT_REPOSITORY  = 'git@github.com:jenkinsci/packaging.git'
-    PACKAGING_GIT_BRANCH      = 'master'
+    PACKAGING_GIT_BRANCH      = '2.387'
```
Comment (timja) [inline Jenkinsfile.d/core/package:86]:
```suggestion
    PACKAGING_GIT_BRANCH      = 'stable-2.387'
```

**Review by timja — COMMENTED** [review]
should be prefixed with stable

**Comment by NotMyFault** [comment]
Looks like they slipped, oops!

**Review by timja — APPROVED** [review]

**Pull request merged**